src/packages/fuelrats_api/rat.py

```diff
@@ -0,0 +1,14 @@
+import aiohttp
```

Comment:
[question] Is this module even used anywhere or a unused artifact of a old implementation?